### PR TITLE
Enable PRC by default in GCP

### PR DESCRIPTION
**Pull request by VenelinMartinov**
This rolls out PlanResourceChange to GCP.

Part of https://github.com/pulumi/pulumi-terraform-bridge/issues/1785

### CODE REVIEW

**Comment by t0yv0** [comment]
Looks like there are some test failures here that may be meaningful.

**Comment by VenelinMartinov** [comment]
Yup, will investigate.

**Comment by VenelinMartinov** [comment]
This should be good to go after https://github.com/pulumi/pulumi-gcp/pull/2305 and the the next bridge release which pulls in https://github.com/pulumi/pulumi-terraform-bridge/pull/2286

File: provider/provider_yaml_test.go [hunk 1]
```diff
@@ -387,18 +387,28 @@ func TestNodePoolGpuAcceleratorPanic(t *testing.T) {
 		},
 		"response": {
 			"properties": {
```
Comment (iwahbe) [inline provider/provider_yaml_test.go:389]: Given that we are testing that we don't panic, we can be pretty lenient on what properties we accept.
```suggestion
			"properties": "*"
```

**Review by iwahbe — APPROVED** [review]
LGTM

File: provider/provider_yaml_test.go (hunk 1 above)
Comment (VenelinMartinov) [reply provider/provider_yaml_test.go]: Yeah, nice

**Pull request merged**